code/datums/diseases/advance/symptoms/sneeze.dm

```diff
@@ -53,8 +53,9 @@
 			if(!suppress_warning)
 				affected_mob.emote("sniff")
 		else
+			affected_mob.emote("sneeze")
 			if(affected_mob.can_spread_airborne_diseases()) //don't spread germs if they covered their mouth
-				affected_mob.infectious_sneeze(active_disease, TRUE, range = spread_range)
+				active_disease.airborne_spread(spread_range = 2, force_spread = TRUE, require_facing = TRUE)
```

Comment:
You don't need the `can_spread_airborne_diseases` check, `airborne_spread` checks it for you

Reply:
Also, I believe you've messed up an argument here:
`ctive_disease.airborne_spread(spread_range = spread_range , force_spread = TRUE, require_facing = TRUE)

Reply:
Oops, good catch!